### PR TITLE
New package: freefall-4.d74aae4e

### PR DESCRIPTION
**Pull request by GiedriusS**

### CODE REVIEW

File: srcpkgs/freefall/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'freefall'
+pkgname="freefall"
+version="4.d74aae4e"
```
Comment (Gottox) [inline srcpkgs/freefall/template]: where does this version come from?

File: srcpkgs/freefall/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+revision=1
+create_wrksrc=yes
+short_desc="Disk protection for HP/DELL machines"
+maintainer="Giedrius Statkevičius <giedriuswork@gmail.com>"
+license="GPL-2"
+homepage="http://www.kernel.org"
+checksum="6fad3e93be9fdb5fbf87ced466961d0c3190614bf5e8efdb0fa20071f4dc06ba"
+distfiles="https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/plain/Documentation/laptops/freefall.c"
```
Comment (Gottox) [inline srcpkgs/freefall/template]: The package will break as soon as this file is updated. Use a specific revision of this file.

File: srcpkgs/freefall/INSTALL.msg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+By default freefall service uses /dev/sda so edit /etc/sv/freefall/run and add
+argument to freefall if you want to use a different device
```
Comment (Gottox) [inline srcpkgs/freefall/INSTALL.msg]: Not a good idea, xbps will overwrite the file on the next update.

File: srcpkgs/freefall/files/freefall/run [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec freefall
```
Comment (Gottox) [inline srcpkgs/freefall/files/freefall/run]: add ${OPTS} to allow custom arguments. See: https://github.com/GiedriusS/void-packages/blob/master/srcpkgs/mongodb/files/mongodb/run for an example

**Comment by xtraeme** [comment]
The run script should be something like this:

```
#!/bin/sh
[ -r conf ] && . ./conf
exec freefall ${OPTS:=/dev/sda}
```

**Pull request closed without merging**